salt/_pillar/metalk8s.py

```diff
@@ -49,12 +51,42 @@ def _load_ca(config_data):
     }
 
 
+def _get_endpoints(config="/etc/kubernetes/admin.conf",
```

Comment:
Don't use defaults like this, they're arbitrary. Values should be passed in from configuration.

Reply:
will fix